### PR TITLE
Feature - adding a README.md translate in Brazilian Portuguese.

### PR DESCRIPTION
**Pull request by Lucbm99**
### Description

I added a README.md translate in Brazilian Portuguese for the documentation has support in many languages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ This project exists thanks to all the [contributors]((https://github.com/globoco
 
 huskyCI is licensed under the [BSD 3-Clause "New" or "Revised" License](https://github.com/globocom/huskyCI/blob/master/LICENSE.md).
 
+*This article can also be read in [Brazilian Portuguese](README-ptBR.md).*
```
Comment (fguisso) [inline README.md:103]: You can put this line at the top of the document.

File: README-ptBR.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+O huskyCI é uma ferramenta de código aberto que orquestra testes de segurança e centraliza todos os resultados em um banco de dados para análises e métricas posteriores. Ele pode realizar análises de segurança estática em Python ([Bandit][Bandit] e [Safety][Safety]), Ruby ([Brakeman][Brakeman]), JavaScript ([Npm Audit][NpmAudit] e [Yarn Audit][YarnAudit]), Golang ([Gosec][Gosec]), Java ([SpotBugs][SpotBugs] mais [Find Sec Bugs][FindSec]), e HCL ([TFSec][TFSec]). Também pode auditar repositórios em busca de segredos como Chaves Secretas da AWS, Chaves SSH Privadas e muitos outros usando [GitLeaks][Gitleaks].
+
+## Como ele funciona?
+
+Os desenvolvedores podem configurar um novo estágio em seus pipelines de CI para verificar vulnerabilidades:
+
+<p align="center"><img src="huskyCI-stage.png"/></p>
+
+Se forem encontrados problemas de segurança no código, informações como a gravidade, a confiança, o arquivo, a linha e muitas outras informações úteis podem ser exibidas, conforme exemplificado:
```
Comment (fguisso) [inline README-ptBR.md:23]: s/gravidade/severidade
s/confiança/confiabilidade

**Review by fguisso — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Lucbm99) [reply README.md]: ok... solved

File: README-ptBR.md (hunk 2 above)
Comment (Lucbm99) [reply README-ptBR.md]: ok... solved

**Pull request closed without merging**